pkg/watermark/fetch/interface.go

```diff
@@ -30,4 +30,6 @@ type Fetcher interface {
 	GetWatermark(offset isb.Offset) wmb.Watermark
 	// GetHeadWatermark returns the latest watermark based on the head offset
 	GetHeadWatermark() wmb.Watermark
+	// GetHeadWMB returns the latest idle WMB from Vn-1 processors
+	GetHeadWMB() wmb.WMB
```

Comment:
In `GetHeadWMB`'s comment, we explicitly call Vn-1 processors; isn’t it true for `GetHeadWatermark` too?